.github/workflows/build-jobs.yaml

```diff
@@ -123,6 +127,15 @@ jobs:
           cd ../build_relwithdebinfo
           ninja ytserver-all
 
+      - name: Build ytsaurus (ya make, relwithdebinfo)
+        if: ${{ inputs.build-query-tracker-ya-make }}
+        run: |
+          mkdir -p ../ya_build
+          cd ../ya_build
+          rm -rf *
+
+          ../ytsaurus/ytsaurus/ya make -T --build=relwithdebinfo  --output=../ya_build ../ytsaurus/ytsaurus/yt/yt
```

Comment:
We build all targets in yt/yt here to save the previous logic (checking that our code can be build by ya make)
https://github.com/ytsaurus/ytsaurus/pull/379/files#diff-0e63f0336811c0261bddff6fc554b7bee0e469a72fd20a1a3bacec95179f7036L52

It also doesn't change the build time too much.